src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.types.ts

```diff
@@ -36,3 +36,7 @@ export interface ThresholdGroupType {
 }
 
 export type AvailableThresholdTypes = 'IgnoreThreshold' | 'FailImmediately'
+
```

Comment:
lets pic types from service file 
```suggestion
export type AvailableThresholdTypes = keyof MetricThresholdCriteriaSpec
```

Reply:
I actually have all these type changes to raise in next PR, let me push all those to this branch itself

Reply:
updated now, please check @deepesh-ui